examples/feeCollection.js

```diff
@@ -0,0 +1,18 @@
+const { Account, Vault } = require('..');
+
+(async () => {
+  const account = new Account({
+    mnemonic: 'lens sell....',
+    network: 'LocalHost',
+  });
+  await account.initialiseReachAccount();
```

Comment:
I don't think this is necessary since it is already called in all internal functions in the SDK